css/properties/line-break.json

```diff
@@ -118,24 +118,22 @@
             ],
             "support": {
               "chrome": {
-                "version_added": "≤83"
+                "version_added": "25"
```

Comment:
Given that `auto` is the default value for this property, I'm not sure what the actual behavior is, so how do we determine whether a browser support this? By checking that we read this value if we set the value?

Otherwise I would expect this value to be supported from Chrome 1.

Reply:
I can see that the default value changed from `normal` to `auto` in Chrome 25, so it's plausible that it's supported from this version: https://github.com/caugner/css-properties/commit/aad719c47873c566d3d7e7b5904a520c41a7c6b4